src/main/java/org/jabref/logic/importer/fileformat/PdfContentImporter.java

```diff
@@ -565,13 +572,16 @@ Optional<BibEntry> getEntryFromPDFContent(String firstpageContents, String lineS
         if (doi != null) {
             entry.setField(StandardField.DOI, doi);
         }
+        if (arXivId != null) {
+            entry.setField(StandardField.EPRINT, arXivId);
+        }
         if (series != null) {
             entry.setField(StandardField.SERIES, series);
         }
         if (volume != null) {
             entry.setField(StandardField.VOLUME, volume);
         }
-        if (number != null) {
+        if (number != null && number.chars().allMatch(Character::isDigit)) {
```

Comment:
```suggestion
        if (number != null) {
```

Maybe that was a fix for some other issue?